projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.scss

```diff
@@ -1,18 +1,25 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
 
 .home-component {
-  max-width: 1280px;
+  max-width: 976px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Let's try to use some the common classes defined for flex utilities directly in the HTML